code/src/components/MainGame.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react'
+import { useSelector, useDispatch} from "react-redux";
+import { fetchGameSteps } from '../reducers/game';
+
+import { useNavigate } from 'react-router-dom';
+import { MainGameStyle, DirectionSection, WinSection, DirectionButton, InfoSection } from "./Style"
+
+const MainGame = () => {
+    const gameObject = useSelector((store) => store.game.gameObject)
+    const username = useSelector((store) => store.game.username)
+    const loading = useSelector((state) => state.ui.loading)
+    const navigate = useNavigate()
+
+    const dispatch = useDispatch()
+
+    const onRestartClick = () => {
+        navigate('/')
+    }
+
+    return (         
+        <div>
+            {!loading && (
+            <MainGameStyle>
+
+                <InfoSection className="nes-container with-title is-centered">
+                <p className="title" tabIndex='0'>User: {username} </p>
+                <p tabIndex='0'>{gameObject.description}</p>
+                </InfoSection>
+
+                <DirectionSection className="nes-container with-title is-centered">
+                {gameObject.actions.length > 0 && (
+                    <div>
+                    {gameObject.actions.map((action) => {
+                        return(
+
+                            <div key={action.direction}>
+                            <p tabIndex='0'>{action.description}</p>
+                        
+                            <DirectionButton
+                                type="button" 
+                                className="nes-btn is-warning"
+                                tabIndex='0'
```

Comment:
This tabIndex though, does not seem to work when I try to go through the page via keyboard. The buttons are not "focused".